packages/prisma/schema.prisma

```diff
@@ -660,7 +660,6 @@ enum AppCategories {
   other
   payment
   video // deprecated, please use 'conferencing' instead
-  web3 // deprecated, we should no longer have any web3 apps
```

Comment:
Getting this build error: The values [web3] on the enum `AppCategories` will be removed. If these variants are still used in the database, this will fail.

Reply:
one easy solution is just marking it as depreceated and not removing it from schema.prisma file.

Reply:
done